tools/cal_scripts/ctd_cal_parser.py

```diff
@@ -56,6 +56,26 @@ def __init__(self):
             'T4': 'CC_T4',
             'T5': 'CC_T5',
         }
+        self.o_series_coefficients_map = {
```

Comment:
We split out the CTD and DO sensors (still need to finalize). Are we sure we want to pull in the oxygen sensor calibrations from the CTD here?

Reply:
I thought we were leaving those in. Hence I put them in the CTD parser. What's the status of this?

Reply:
You're correct - we need to leave in place until the DO sensor is up and the CTD is using the new stream w/o DO values.